ScratchWikiSkinTemplate.php

```diff
@@ -103,6 +103,37 @@ public function execute() {
 					</div>
 				</form>
 			</li>
+			<?php
+			//  Start dropdown here so we know its output before it saves to the page, and we can use it earlier.
+			$swsDropdownItems = "";
+			foreach ($this->data['personal_urls'] as $key => $tab) { 
+					if (!array_key_exists('href', $tab)) {
+						if (!array_key_exists('class', $tab)) {
+							$tab['class'] = '';
+						}
+						
+						$tab['class'] .= 'no-link';
+					}
+					if(isset($tab["class"]) && gettype($tab["class"]) == "array" && $tab["class"][0] == "mw-echo-alert"){
```

Comment:
a comment here to explain what the significance of all this is would be good

Reply:
There already is a comment

Reply:
A comment on the specific line I highlighted would be good to explain what the purpose of this if statement is

Reply:
A new comment has been added